docs/network-flow-visibility.md

```diff
@@ -202,30 +203,69 @@ helm install theia build/charts/theia -n flow-visibility --create-namespace \
   --set=clickhouse.storageSize="2Gi"
 ```
 
-The ClickHouse TCP service is used by the Flow Aggregator. If you have changed the TCP port,
-please update the `databaseURL` following
-[Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1).
+The ClickHouse TCP Service is used by the Flow Aggregator. If you have changed
+the TCP port, please update the `clickHouse.databaseURL` in the Flow Aggregator
+Helm Chart values.
+
+The ClickHouse credentials are used in the Flow Aggregator. If you have changed
+the ClickHouse credentials, please update the `clickHouse.connectionSecret` in the
+Flow Aggregator Helm Chart values.
+
+Please refer to the Flow Aggregator Helm Chart
+[README](https://github.com/antrea-io/antrea/blob/main/build/charts/flow-aggregator/README.md)
+and [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1)
+to learn more about ClickHouse configuration options in Flow Aggregator.
+
+##### ClickHouse Cluster
+
+In v0.1, Theia deploys ClickHouse in a non-cluster mode, which cannot switch to
+ClickHouse cluster directly. You need to uninstall ClickHouse, delete related
+data if you use PersistenVolume and redeploy it when upgrading or downgrading
+between v0.1 and v0.2.
+
+Starting with v0.2, Theia supports ClickHouse clustering in lieu of the
+non-clustered mode supported in v0.1. A ClickHouse cluster consists of one or more
+shards. Shards refer to the servers that contain different parts of the data. You
+can deploy multiple shards to scale the cluster horizontally. Each shard consists
+of one or more replica hosts. Replicas indicate storing the same data on multiple
+Pods to improve data availability and accessibility. You can deploy multiple
+replicas in a shard to ensure reliability.
+
+By default, Theia deploys a 1-shard-1-replica ClickHouse cluster and a 1-replica
+ZooKeeper. To scale the ClickHouse cluster, please set `clickhouse.cluster.shards`
+and `clickhouse.cluster.replicas` per your requirement.
+
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org/) for storing
+replicas meta information. When scaling the ClickHouse cluster, please set
+`clickhouse.cluster.installZookeeper.replicas` to at least 3 to ensure fault
+tolerance. Each replica is expected to be deployed on a different Node. To
+leverage a user-provided ZooKeeper cluster, please refer to the
+[ZooKeeper setup instructions for ClickHouse](https://github.com/Altinity/clickhouse-operator/blob/master/docs/zookeeper_setup.md)
+and set `clickhouse.cluster.zookeeperHosts` to your ZooKeeper hosts.
+
+When stopping the ClickHouse cluster, please ensure it is stopped before ZooKeeper,
+as ClickHouse relies on ZooKeeper to stop correctly. You can run the following
+command to stop the ClickHouse cluster.
```

Comment:
What's the purpose of this paragraph? Is it for the users who want to switch their clickhouse back to non-cluster mode?

Reply:
No. Just for any user who wants to stop ClickHouse. I notice that if users stop the ClickHouse cluster with our manifests directly, it cannot stop correctly.

Reply:
Thanks for explanation. Do you think we should combine this paragraph with L157 or move it before L157?

Reply:
I think it makes sense to mention this in the Deployment Section. But as this paragraph introduces ZooKeeper, I'm a little bit worry about adding these to L157 without the context will be confused. I try to update the Deployment Section. Please take a look if it is understandable.

For your question about the `helm uninstall`, yes, it is true.

Reply:
It looks good now. Thanks.